mqtt-streaming/src/main/scala/akka/stream/alpakka/mqtt/streaming/impl/ClientState.scala

```diff
@@ -183,17 +189,17 @@ import scala.util.{Failure, Success}
         ConnectReceived(
           connect,
           connectData,
-          Vector.empty,
           queue,
+          Vector.empty,
           data.consumerPacketRouter,
           data.producerPacketRouter,
           data.subscriberPacketRouter,
           data.unsubscriberPacketRouter,
           data.settings
         )
       )
-    case (_, prl: PublishReceivedLocally) if data.stash.size < data.settings.maxClientConnectionStashSize =>
-      disconnected(data.copy(stash = data.stash :+ prl))
+    case (_, e) =>
+      disconnected(data.copy(stash = data.stash :+ e))
```

Comment:
I'm unsure why this was previously constrained in terms of what messages should be stashed. We could have easily dropped some accidentally.